rep-2001.rst

```diff
@@ -35,81 +35,599 @@ End-user entry points
+
+
+Desktop
+"""""""
 
 ::
 
   - desktop:
-      extends: [ros_base]
-      packages: [angles, demos, depthimage_to_laserscan, example_interfaces,
-                 examples, joystick_drivers, laser_geometry,
-                 navigation_msgs, pcl_conversions, realtime_support,
-                 resource_retriever, rviz, teleop_twist_joy,
-                 teleop_twist_keyboard, tlsf, turtlesim, vision_opencv]
+      extends:  [ros_base]
+      packages: [action_tutorials, angles, composition,
+                 demo_nodes_cpp, demo_nodes_cpp_native,
+                 demo_nodes_py, depthimage_to_laserscan,
+                 dummy_map_server, dummy_robot_bringup,
+                 dummy_sensors,
+                 examples_rclcpp_minimal_action_client,
+                 examples_rclcpp_minimal_action_server,
+                 examples_rclcpp_minimal_client,
+                 examples_rclcpp_minimal_composition,
+                 examples_rclcpp_minimal_publisher,
+                 examples_rclcpp_minimal_service,
+                 examples_rclcpp_minimal_subscriber,
+                 examples_rclcpp_minimal_timer,
+                 examples_rclpy_executors,
+                 examples_rclpy_minimal_action_client,
+                 examples_rclpy_minimal_action_server,
+                 examples_rclpy_minimal_client,
+                 examples_rclpy_minimal_publisher,
+                 examples_rclpy_minimal_service,
+                 examples_rclpy_minimal_subscriber, image_tools,
+                 intra_process_demo, joy, lifecycle, logging_demo,
+                 pcl_conversions, pendulum_control, pendulum_msgs,
+                 quality_of_service_demo_cpp,
+                 quality_of_service_demo_py, rqt_common_plugins,
+                 rviz2, rviz_default_plugins, teleop_twist_joy,
+                 teleop_twist_keyboard, tlsf, tlsf_cpp,
+                 topic_monitor, turtlesim]
+
+
+Eloquent Elusor (November 2019 - November 2020)
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
+ROS Core
+""""""""
 
-Institution-specific and robot-specific
-'''''''''''''''''''''''''''''''''''''''
+::
 
-Please see REP 108 [1]_ for discussion of institution-specific
-metapackages.
+  - ros_core:
+      packages: [ament_cmake, ament_cmake_auto, ament_cmake_gmock,
+                 ament_cmake_gtest, ament_cmake_pytest,
+                 ament_cmake_ros, ament_index_cpp,
+                 ament_index_python, ament_lint_auto,
+                 ament_lint_common, class_loader, common_interfaces,
+                 pluginlib, rcl_lifecycle, rclcpp, rclcpp_lifecycle,
+                 rclpy, ros2action, ros2component, ros2doctor,
+                 ros2interface, ros2launch, ros2lifecycle, ros2msg,
+                 ros2multicast, ros2node, ros2param, ros2pkg,
+                 ros2run, ros2service, ros2srv, ros2topic,
+                 ros_environment, rosidl_default_generators,
+                 rosidl_default_runtime, sros2, sros2_cmake]
```

Comment:
This seems to be missing the section saying "and at least one of the following `rmw_implementation`".

Reply:
Fixed in https://github.com/ros-infrastructure/rep/pull/327/commits/59d0c03654ae9b95b1470150747a97bcb44be209.